lib/resolver-compat.js

```diff
@@ -291,6 +291,8 @@ function resolverFromOptions(vm, options, override, compiler) {
 
 	const builtins = genBuiltinsFromOptions(vm, builtinOpt, mockOpt, override);
 
+	const pathContext = options.pathContext || (() => context);
```

Comment:
I think it would be better to pass the `pathContext` function through the `context` option so that the `context` option then allows to be `"host" | "sandbox" | pathContextCallback`.

Reply:
Yeah, that's definitely a cleaner option. Will change.